enarx-auto-request

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+import enarxbot
+import random
+
+github = enarxbot.connect()
+org = github.get_organization("enarx")
+
+# Auto-assign for all PRs in the org.
+for issue in github.search_issues(f"org:enarx is:pr is:public is:open -is:draft"):
+    repo = issue.repository
+    pr = issue.as_pull_request()
+
+    # First, put Nathaniel on all code reviews as code owner.
+    npmccallum = github.get_user("npmccallum")
+
+    # Construct sets of reviewers. Exclude both Nathaniel and the PR author.
+    suggested = {s.reviewer for s in enarxbot.Suggestion.query(github, repo, pr)} - {npmccallum} - {pr.user}
+    reviews = {t.name: t for t in org.get_teams()}['reviews']
+    reviews_team = {r for r in reviews.get_members()} - {npmccallum} - {pr.user}
+
+    # Figure out if currently requested reviewers include Nathaniel and two
+    # reviewers from the Reviews team.
+    requested_reviewers = {r for r in pr.get_review_requests()[0]}
+    requested_review_team = requested_reviewers & reviews_team
+    if npmccallum in requested_reviewers and len(requested_review_team) >= 2:
+        continue
+
+    # If there are valid suggested reviewers, and fewer than two people
+    # from the reviews team requested for review, grab a suggested reviewer.
+    suggested_reviewer = None
+    suggested_reviewer_login = []
+    if len(requested_review_team) < 2 and len(suggested) > 0:
+        suggested_reviewer = random.sample(suggested, 1)[0]
+        suggested_reviewer_login += [suggested_reviewer.login]
+
+    # If there are no reviewers requested from the reviews team, grab one.
+    reviews_team = reviews_team - {suggested_reviewer}
+    random_reviewer_login = []
+    if len(requested_review_team)  == 0:
+        random_reviewer_login += [random.sample(reviews_team, 1)[0].login]
+
+    to_request = [npmccallum.login] + suggested_reviewer_login + random_reviewer_login +  [r.login for r in requested_reviewers]
```

Comment:
`to_request` ends up being a list instead of a set. This means that some users appear twice.